search/views.py

```diff
@@ -194,3 +203,37 @@ def subsq(a, b, m, n):
         return subsq(a, b, m - 1, n - 1)
     # If last characters are not matching
     return subsq(a, b, m, n - 1)
+
+
+def search_results_from_sites(request):
+    keyword = request.GET['q']
+    keyword.replace(' ', '+')
+    res = requests.get('https://google.com/search?q=' + keyword)
```

Comment:
If we want to get results for a particular keyword say `linear search` from Wikipedia, then, the query should be `linear search site:wikipedia.org` 

Kindly take a look into this.